scheds/rust/scx_p2dq/src/lib.rs

```diff
@@ -53,6 +53,11 @@ pub struct SchedulerOpts {
     #[clap(short = 'd', long, action = clap::ArgAction::SetTrue)]
     pub dispatch_pick2_disable: bool,
 
+    /// Enables pick2 load balancing on the dispatch path when LLC utilization is under the
+    /// specified utilization.
+    #[clap(long, default_value = "75", value_parser = clap::value_parser!(u64).range(0..99))]
```

Comment:
Is it worth allowing 100 to make this path always true?

Reply:
Good call